include/oneapi/dpl/pstl/hetero/dpcpp/execution_sycl_defs.h

```diff
@@ -16,21 +16,14 @@
 #ifndef _ONEDPL_execution_sycl_defs_H
 #define _ONEDPL_execution_sycl_defs_H
 
-#include <CL/sycl.hpp>
 #include "../../onedpl_config.h"
 #include "../../execution_defs.h"
+
+#include "sycl_defs.h"
```

Comment:
```sycl_defs``` must be included prior to ```execution_defs```, because it includes ```execution_sycl_defs```, which may use functionality of ```sycl_defs```. I suggest including of ```sycl_defs``` before all oneDPL headers.